app.rb

```diff
@@ -4,17 +4,57 @@
 
 # First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
   # Explain how the data is structured
+  It's categorized by apartments and tenants. It has string value sand numerical values.
   # What are the properties for each of the two types of hashes
 
+
 # Use enumerables to -
   # Print all the addresses for the apartments
+      appartments[:address]
+
+      apartments.each do |apartment|
+        puts apartment[:address]
+      end
+
   # Print all the names for tenants
+      tenants[:name]
+
+      tenants.each do [tenant]
+        puts tenant[:name]
+      end
+
+
   # Print only apartments that are less then 700 in rent
+  cheaper_apartments = apartments.select do |apartment|
+      apartment[:monthly_rent] < 700
+      end
+
+
   # Print only tenants that are over the age of 44
+  44_tenants = tenants.select do [tenant]
```

Comment:
There's a syntax error here, after the `do`, we use *pipes * like so: `| tenant |` to specify the enumerator variable.